tic_tac_toe/src/board_test.py

```diff
@@ -0,0 +1,45 @@
+from board import Board
+
+
+def update_column(board):
+    board.values[board.convert(3, 1)] = "X"
+    board.values[board.convert(3, 2)] = "X"
+    board.values[board.convert(3, 3)] = "X"
+
+def update_diagonal(board):
+    board.values[board.convert(3, 3)] = "X"
+    board.values[board.convert(2, 2)] = "X"
+    board.values[board.convert(1, 1)] = "X"
+
+def update_row(board):
+    board.values[board.convert(1, 3)] = "X"
+    board.values[board.convert(2, 3)] = "X"
+    board.values[board.convert(3, 3)] = "X"
+
+def validate_move_update(board):
+    board.values[0] = "X"
+
+def test_column(board):
+    update_column(board)
```

Comment:
Insufficient test cases covered here.  Please cover these (add a new function for each):
* when the column has None in one of the entry and X's and O's in the other
* when the column has None in all of the entries
* when the column has all X's
* when the column has a mix of X's and O's